utils/fasm/src/fasm.h

```diff
@@ -97,6 +101,14 @@ class FasmWriterVisitor : public NetlistVisitor {
       std::map<const t_pb_type*, Parameters> parameters_;
 
       std::map<const std::string, std::string> tags_;
+
+      vtr::interned_string fasm_lut;
+      vtr::interned_string fasm_features;
+      vtr::interned_string fasm_params;
+      vtr::interned_string fasm_prefix;
+      vtr::interned_string fasm_placeholders;
+      vtr::interned_string fasm_type;
+      vtr::interned_string fasm_mux;
```

Comment:
Is it possible to allocate these at compile time and make these constants?

Reply:
I don't think so, and I don't think it would save enough time to be worth the effort.